src/Core/Domain/Product/QueryResult/ProductForEditing.php

```diff
@@ -51,22 +51,37 @@ class ProductForEditing
      */
     private $categoriesInformation;
 
+    /**
+     * @var ProductPricesInformation
+     */
+    private $pricesInformation;
+    /**
+     * @var int
+     */
+    private $taxRulesGroupId;
+
     /**
      * @param int $productId
      * @param bool $active
      * @param ProductBasicInformation $basicInformation
      * @param ProductCategoriesInformation $categoriesInformation
+     * @param ProductPricesInformation $pricesInformation
+     * @param int $taxRulesGroupId
      */
     public function __construct(
         int $productId,
         bool $active,
         ProductBasicInformation $basicInformation,
-        ProductCategoriesInformation $categoriesInformation
+        ProductCategoriesInformation $categoriesInformation,
+        ProductPricesInformation $pricesInformation,
+        int $taxRulesGroupId
```

Comment:
this could be improved later depending on needs. For example we can have a DTO if we need a name or something more from taxRulesGroup.